Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/Tests/id_set.json

```diff
@@ -0,0 +1,21 @@
+{
+    "scripts": [],
+    "playbooks": [],
+    "integrations": [],
+    "TestPlaybooks": [],
+    "Classifiers": [],
+    "Dashboards": [],
+    "IncidentFields": [],
+    "IncidentTypes": [],
```

Comment:
Does this file is required? 
This seems empty..

Reply:
We have deleted this file